genlayer_py/types/transactions.py

```
@@ -413,17 +414,15 @@ def _decode_input_data(self) -> Union[DecodedDeployData, DecodedCallData, None]:
                 }
 
             else:
-                print(
+                logging.warning(
                     "[decode_input_data] Unexpected RLP array length:",
                     len(rlp_decoded_array),
                     rlp_decoded_array,
                 )
```

Pylint (3.3.7)

[error] Too many arguments for logging format string

(E1205)